src/lib.rs

```diff
@@ -1,8 +1,11 @@
 #![feature(plugin_registrar, box_syntax)]
 #![feature(rustc_private, core, collections)]
-#![feature(num_bits_bytes)]
+#![feature(num_bits_bytes, iter_arith)]
 #![allow(unknown_lints)]
 
+#[allow(dead_code)]
+fn main() { println!("What are you doing? Don't run clippy as an executable"); }//~ ERROR: dummy error pattern
```

Comment:
with a missing main, the dogfood `run_pass` trick doesn't work. I can remove the comment though